starter-code/exBook/views/place/show.ejs

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Google Maps | Express</title>
+    <link rel='stylesheet' href='/stylesheets/style.css' />
+</head>
+
+<body>
+    <h3 id="restaurantName" data-id="<%= restaurant.id %>">
```

Comment:
There's no restaurant in this application.

![Copy-Paste](http://i2.wp.com/nikkisiixx.com/wp-content/uploads/2015/10/copypaste-copypaste-everywhere.jpg?resize%5Cu003d600%2C327)